core/ir/aarch64/codec.txt

```diff
@@ -1350,232 +1368,232 @@ x001111001000010xxxxxxxxxxxxxxxx     scvtf     d0 : wx5 scale
+0010111100xxxxxx100001xxxxxxxxxx  n     sqshrun   d0 : d5 immhb
+0110111100xxxxxx100001xxxxxxxxxx  n     sqshrun2  q0 : q5 immhb
+0111111101xxxxxx010001xxxxxxxxxx  n     sri       d0 : d5 immhb
+0x1011110xxxxxxx010001xxxxxxxxxx  n     sri      dq0 : dq5 sd_sz immhb
+0000111000110000110010xxxxxxxxxx  n     fmaxnmv   h0 : d5
+0100111000110000110010xxxxxxxxxx  n     fmaxnmv   h0 : q5
+0110111000110000110010xxxxxxxxxx  n     fmaxnmv   s0 : q5
+0000111010110000110010xxxxxxxxxx  n     fminnmv   h0 : d5
+0100111010110000110010xxxxxxxxxx  n     fminnmv   h0 : q5
+0110111010110000110010xxxxxxxxxx  n     fminnmv   s0 : q5
+0101111011111001110110xxxxxxxxxx  n     frecpe     h0 : h5
+0101111010100001110110xxxxxxxxxx  n     frecpe     s0 : s5
+0101111011100001110110xxxxxxxxxx  n     frecpe     d0 : d5
+0x0011101x100001110110xxxxxxxxxx  n     frecpe    dq0 : dq5 bd_sz
+0101111011111001111110xxxxxxxxxx  n     frecpx     h0 : h5
+0101111010100001111110xxxxxxxxxx  n     frecpx     s0 : s5
+0101111011100001111110xxxxxxxxxx  n     frecpx     d0 : d5
+0111111011111001110110xxxxxxxxxx  n     frsqrte    h0 : h5
+0111111010100001110110xxxxxxxxxx  n     frsqrte    s0 : s5
+0111111011100001110110xxxxxxxxxx  n     frsqrte    d0 : d5
+0x1011101x100001110110xxxxxxxxxx  n     frsqrte   dq0 : dq5 bd_sz
+0000111100xxxxxx100011xxxxxxxxxx  n     rshrn      d0 : d5 immhb
+0100111100xxxxxx100011xxxxxxxxxx  n     rshrn2     q0 : q5 immhb
+0x001110xx110000001110xxxxxxxxxx  n     saddlv    dq0 : dq5 bhsd_sz
+0101111000100000011110xxxxxxxxxx  n     sqabs     b0 : b5
+0101111001100000011110xxxxxxxxxx  n     sqabs     h0 : h5
+0101111010100000011110xxxxxxxxxx  n     sqabs     s0 : s5
+0101111011100000011110xxxxxxxxxx  n     sqabs     d0 : d5
+0x001110xx100000011110xxxxxxxxxx  n     sqabs     dq0 : dq5 bhsd_sz
+0111111011100000100110xxxxxxxxxx  n     cmle       d0 : d5
+0x101110xx100000100110xxxxxxxxxx  n     cmle      dq0 : dq5 bhsd_sz
+0111111011111000110110xxxxxxxxxx  n     fcmle      h0 : h5
+0111111010100000110110xxxxxxxxxx  n     fcmle      s0 : s5
+0111111011100000110110xxxxxxxxxx  n     fcmle      d0 : d5
+0x1011101x100000110110xxxxxxxxxx  n     fcmle     dq0 : dq5 sd_sz
+0101111011111000111010xxxxxxxxxx  n     fcmlt      h0 : h5
+0101111010100000111010xxxxxxxxxx  n     fcmlt      s0 : s5
+0101111011100000111010xxxxxxxxxx  n     fcmlt      d0 : d5
+0x0011101x100000111010xxxxxxxxxx  n     fcmlt     dq0 : dq5 sd_sz
+00011110111xxxxx001000xxxxx10000  w     fcmpe         : h5 h16
+0001111011100000001000xxxxx11000  w     fcmpe         : h5
+00011110001xxxxx001000xxxxx10000  w     fcmpe         : s5 s16
+0001111000100000001000xxxxx11000  w     fcmpe         : s5
+00011110011xxxxx001000xxxxx10000  w     fcmpe         : d5 d16
+0001111001100000001000xxxxx11000  w     fcmpe         : d5
+0111111001100001011010xxxxxxxxxx  n     fcvtxn     s0 : d5
+0010111001100001011010xxxxxxxxxx  n     fcvtxn     d0 : q5
+0110111001100001011010xxxxxxxxxx  n     fcvtxn2    q0 : q5
```

Comment:
The only simple test to add would be encoding some of these opcodes and querying the eflags field vs hardcoded expectations -- still, maybe worth adding a few such sanity checks to api.ir?

Reply:
Noted. Will do this as part of https://github.com/DynamoRIO/dynamorio/issues/5025.